### PR TITLE
REF: upgrade eslint, prettier, fix errors, reformat everything 💄

### PR DESCRIPTION
**Pull request by limpbrains**
Local rebased version of  #1168

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ucrwkz99h1u9wauw7v62we7mmg

[download apk](http://filestorage.bluewallet.io:1488/1591217033.apk)

**Comment by Overtorment** [comment]
bueno.

File: screen/lnd/scanLndInvoice.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ export default class ScanLndInvoice extends React.Component {
   };
 
   async pay() {
-    if (!this.state.hasOwnProperty('decoded')) {
+    if ('decoded' in this.state) {
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js:211]: found that `!` is lost here only after merge. will try to find more such cases and fix it in master, don't really want to unmerge

Comment (limpbrains) [reply screen/lnd/scanLndInvoice.js:211]: OMG sorry about that
I've walked through the patch and couldn't find more errors like this

**Pull request merged**